### ISSUE TITLE
Add 'nightly' tests to test against latest version of napari

### ISSUE THREAD

**Issue by lukasz-migas** [lukasz-migas/napari-plot#117]
Would be good to test against various versions of napari + latest version

**Comment by pnm4sfix** [comment]
Hi @lukasz-migas, great plugin! Any planned further developments against the latest versions of napari?

**Comment by lukasz-migas** [comment]
Yes! I actually recently picked up this project again and trying to get it working.
There have been a lot of changes and improvements to napari lately so it might take me a while but it's coming.
